package_policies/R8.md

```diff
@@ -0,0 +1,7 @@
+**R8.** Each package should provide pre-processor macros for version
```

Comment:
This will need to be R9 or R10 now that we have an R8.

Reply:
fixed now [and squashed/rebased to fix merge conflicts]